### PR TITLE
Blockchain tests initial checkin

### PR DESCRIPTION
**Pull request by aboimpinto**

### CODE REVIEW

File: src/NeoSharp.Core/Cryptography/NativeCrypto.cs [hunk 1]
```diff
@@ -33,50 +43,67 @@ public override byte[] AesEncrypt(byte[] data, byte[] key)
...
+        /// <summary>
+        /// Decrypt using ECB
+        /// </summary>
+        /// <param name="data">Data</param>
+        /// <param name="key">Key</param>
+        /// <returns>Bytearray</returns>
         public override byte[] AesDecrypt(byte[] data, byte[] key)
         {
-            using (var aes = Aes.Create())
+            using (Aes aes = Aes.Create())
```
Comment (shargon) [inline src/NeoSharp.Core/Cryptography/NativeCrypto.cs:54]: change again?

**Comment by shargon** [comment]
@aboimpinto please review, is like if some of your last changes will be reverted

**Review by shargon — CHANGES_REQUESTED** [review]
@aboimpinto please review, is like if some of your last changes will be reverted

**Comment by aboimpinto** [comment]
those are independent merges ... after the first been merge .. the second need to resolve the conflicts, therefore they can be merge separately

**Comment by aboimpinto** [comment]
i did't ... are just different branches ..

**Comment by aboimpinto** [comment]
this PR was merged with the PR: https://github.com/CityOfZion/neo-sharp/pull/232

**Pull request closed without merging**